zebra-scan/src/storage.rs

```diff
@@ -19,6 +19,10 @@ pub use db::{SaplingScannedResult, SaplingScanningKey};
 
 use self::db::ScannerWriteBatch;
 
+/// We insert an empty results entry to the database every this interval for each stored key,
+/// so we can track progress.
+const INSERT_CONTROL_INTERVAL: u32 = 1_000;
```

Comment:
Does not need to be the same as `INFO_LOG_INTERVAL` but can be, to make things a bit simpler.

Reply:
It can take 9 hours to scan the most shielded-heavy 100,000 blocks, which is 4 blocks per second.

So I would recommend keeping this around 1000, regardless of the log interval.